site/docs/spec.md

```diff
@@ -170,6 +170,7 @@ Partition specs capture the transform from table data to partition values. This
 | **`month`**       | Extract a date or timestamp month, as months from 1970-01-01 | `date`, `timestamp(tz)`                                                                                   | `int`       |
 | **`day`**         | Extract a date or timestamp day, as days from 1970-01-01     | `date`, `timestamp(tz)`                                                                                   | `date`      |
 | **`hour`**        | Extract a timestamp hour, as hours from 1970-01-01 00:00:00  | `timestamp(tz)`                                                                                           | `int`       |
+| **`alwaysNull`**  | Always produces `null` (the void transform)                  | Any                                                                                                       | `void`      |
```

Comment:
The transform name for the spec is `void`, not `alwaysNull`. That's just the name we use in Java where `void` is a reserved keyword.

Reply:
Thanks for the comments. Fixed.